cogs5e/models/character.py

```diff
@@ -479,6 +523,34 @@ def get_sheet_embed(self):
                                   f"Please run !update.")
 
         return embed
+    
+    def active_embed(self):
+        """Creates an embed to be displayed when the active character is checked"""
+        embed = EmbedWithCharacter(self)
+        embed.title = self.name
+        urls = {"beyond": "https://ddb.ac/characters/",
+                "dicecloud": "https://dicecloud.com/character/",
+                "google": "https://docs.google.com/spreadsheets/d/"}
+        sheeturl = urls[self.sheet_type] + self.upstream_id
+        desc = f"Your current active character is {active_character.name}. " \
```

Comment:
`active_character` is undefined here, did you mean `self`?

Reply:
Yes
```suggestion
        desc = f"Your current active character is {self.name}. " \
```